csv/ParameterDefs.csv

```diff
@@ -2543,7 +2543,7 @@ TO-DO,METBK,met_relwind_speed,met_relwind_speed,PD3073,function,,float32,,m s-1,
 TO-DO,METBK,met_timeflx,met_timeflx,PD3074,function,,float32,,seconds since 1900-01-01T00:00:00Z,-9999999,Hourly Averaged Timestamp,4,,PFID150,"{""timestamp"":""PD7""}",,,TIMEFLX-AUX,"TIMEFLX-AUX, the timestamps corresponding to the hourly averaged METBK data products. The units of the timestamps are seconds since 01-01-1900. The timestamp values are selected to be at the midpoint of the bin intervals, starting half an hour after the timestamp of the first data record to be processed. For example, if the first data record for 30 days of data is at 4:45 AM on a given day, the first timeflx stamp will be at 5:15 AM on that day, and all succeeding timestamps for the rest of the data will all be at 15 minutes past the hour.",,,
 TO-DO,METBK,met_netsirr,met_netsirr,PD3075,function,,float32,,W m-2,-9999999,Net Shortwave Irradiance,4,,PFID151,"{""shortwave_down"":""PD1058""}",,,NETSIRR_L2,"NETSIRR_L2, the OOI core data product net shortwave radiation in the downward direction, for the METBK instrument.",,Science Data,L2
 TO-DO,METBK,met_rainrte,met_rainrte,PD3076,function,,float32,,mm hr-1,-9999999,Rain Rate,4,,PFID152,"{""cumulative_precipitation"":""PD1055"", ""timestamp"":""PD7""}",,,RAINRTE_L2,"RAINRTE_L2, the OOI core data product rain rate, for the METBK instrument. The DPS requires that the output data be hourly; METBK is set up to give roughly one data record per minute for the data needed to calculate RAINRTE.",,Science Data,L2
-PCO2A,"METBK, PCO2A",met_salsurf,met_salsurf,PD3077,function,,float32,,1E-03,-9999999,Sea Surface Practical Salinity,4,,PFID153,"{""cond"":""dpi_CONDSRF_L1"", ""tC_sea"":""dpi_TEMPSRF_L1"", ""ztmpwat"":""CC_depth_of_conductivity_and_temperature_measurements_m""}",SALSURF,sea_surface_salinity,SALSURF_L2,Salinity is generally defined as the concentration of dissolved salt in a parcel of seawater. Practical Salinity is a more specific unitless quantity calculated from the conductivity of seawater and adjusted for temperature and pressure. It is approximately equivalent to Absolute Salinity (the mass fraction of dissolved salt in seawater) but they are not interchangeable.,Sea Surface Practical Salinity,Science Data,L2
+PCO2A,"METBK, PCO2A",met_salsurf,met_salsurf,PD3077,function,,float32,,1.00E-03,-9999999,Sea Surface Practical Salinity,4,,PFID153,"{""cond"":""dpi_CONDSRF_L1"", ""tC_sea"":""dpi_TEMPSRF_L1"", ""ztmpwat"":""CC_depth_of_conductivity_and_temperature_measurements_m""}",SALSURF,sea_surface_salinity,SALSURF_L2,Salinity is generally defined as the concentration of dissolved salt in a parcel of seawater. Practical Salinity is a more specific unitless quantity calculated from the conductivity of seawater and adjusted for temperature and pressure. It is approximately equivalent to Absolute Salinity (the mass fraction of dissolved salt in seawater) but they are not interchangeable.,Sea Surface Practical Salinity,Science Data,L2

```

Comment:
Sorry, this one is an artifact of using Excel to make some of the changes.  I can reverse it if desired.

Reply:
Since Chris suggested a CF standard name for the backscatter beta, I took that opportunity to remove the extra decimal and zeros from this line.  So I'm resolving this comment.